### PR TITLE
Latest sources for iOS Push Starter

### PR DESCRIPTION
**Pull request by sam-almighty**

### CODE REVIEW

**Review by gee4vee — APPROVED** [review]

**Comment by gee4vee** [comment]
Can you add `.DS_Store` to .gitignore please? We don't want those files in source control.

**Comment by sam-almighty** [comment]
> Can you add `.DS_Store` to .gitignore please? We don't want those files in source control.

Done. Included all the basic ignores.

**Review by gee4vee — APPROVED** [review]

**Review by gee4vee — CHANGES_REQUESTED** [review]
Found some changes needed.

1. Delete .DS_Store file.
2. Do not delete the `Podfile`. We keep these in our mobile projects for others to use. Is there a particular reason why you deleted it? You can delete the `Podfile.lock` though.
3. Do not include any `xcuserdata` files in source control. Those are for local environments only.
4. See additional comments below.

File: ios-push-app.xcodeproj/xcuserdata/nostanle.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gee4vee) [inline ios-push-app.xcodeproj/xcuserdata/nostanle.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:1]: Typically we do not put any of the `xcuserdata` files in source control. Can you remove them?

File: ios-push-app.xcodeproj/xcuserdata/nostanle.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 2]
```diff
@@ -2,7 +2,13 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (gee4vee) [inline ios-push-app.xcodeproj/xcuserdata/nostanle.xcuserdatad/xcschemes/xcschememanagement.plist:2]: Do not include `xcshareddata` either.

File: ios-push-app/Base.lproj/LaunchScreen.storyboard [hunk 3]
```diff
@@ -1,22 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (gee4vee) [inline ios-push-app/Base.lproj/LaunchScreen.storyboard:1]: We actually need to keep the module name as `iosapp`. The reason is that our code generator does not know about the specific skit name and expects only `iosapp`. We are making a new generation of code generator that will be better, but for now we are avoiding making major changes to the existing one.

File: ios-push-app/CustomUrlButton.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
+//
```
Comment (gee4vee) [inline ios-push-app/CustomUrlButton.swift:1]: Keep the folder name as `iosapp` (see above).

File: ios-push-app/Info.plist [hunk 5]
```diff
@@ -3,40 +3,42 @@
 <plist version="1.0">
```
Comment (gee4vee) [inline ios-push-app/Info.plist:3]: Keep the folder name as `iosapp` (see above).

File: ios-push-app/SceneDelegate.swift [hunk 6]
```diff
@@ -0,0 +1,53 @@
+//
```
Comment (gee4vee) [inline ios-push-app/SceneDelegate.swift:1]: Keep the folder name as `iosapp` (see above).

File: ios-push-app/ViewController.swift [hunk 7]
```diff
@@ -0,0 +1,47 @@
+//
```
Comment (gee4vee) [inline ios-push-app/ViewController.swift:1]: Keep the folder name as `iosapp` (see above).

File: iosappTests/Info.plist [hunk 8]
```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gee4vee) [inline iosappTests/Info.plist:1]: Keep the folder name as `iosappTests` (see above).

File: iosappTests/iosappTests.swift [hunk 9]
```diff
@@ -1,33 +0,0 @@
-//
```
Comment (gee4vee) [inline iosappTests/iosappTests.swift:1]: Keep the folder name as `iosappTests` (see above).

File: iosappUITests/Info.plist [hunk 10]
```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gee4vee) [inline iosappUITests/Info.plist:1]: Keep the folder name as `iosappUITests` (see above).

File: iosappUITests/iosappUITests.swift [hunk 11]
```diff
@@ -1,33 +0,0 @@
-//
```
Comment (gee4vee) [inline iosappUITests/iosappUITests.swift:1]: Keep the folder name as `iosappUITests` (see above).

File: ios-push-app/AppDelegate.swift [hunk 12]
```diff
@@ -0,0 +1,37 @@
+//
```
Comment (gee4vee) [inline ios-push-app/AppDelegate.swift:1]: Keep the folder name as `iosapp` (see above).

File: ios-push-app/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 13]
```diff
@@ -0,0 +1,98 @@
+{
```
Comment (gee4vee) [inline ios-push-app/Assets.xcassets/AppIcon.appiconset/Contents.json:1]: Keep the folder name as `iosapp` (see above).

File: ios-push-app/Assets.xcassets/Contents.json [hunk 14]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (gee4vee) [inline ios-push-app/Assets.xcassets/Contents.json:1]: Keep the folder name as `iosapp` (see above).

**Comment by youming-lin** [comment]
@sam-almighty Can you take a look at the requested changes? We are looking to deprecate the old mobile starter kits in favor of these new replacements; this repo is one of the replacements.

**Comment by gee4vee** [comment]
@sam-almighty any update on when you can get to this?

**Comment by AnanthaKrish** [comment]
I am looking  into this, probably by tomorrow we can close this.

**Comment by AnanthaKrish** [comment]
Please find the new PR here - https://github.com/IBM/ios-push-app/pull/2

**Comment by gee4vee** [comment]
Can we close this one in favor of the other? I've added comments to it.

**Pull request closed without merging**